lib/scripts/clone.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -ex #TODO x #ONLY FOR DEV
+
+repo_root=$1
```

Comment:
not tested:
if [ "$1" = "--use-submodules" ]; then
  $use_submodules=1
  shift
fi